Expose delay options

mmatous opened the pull request:
This adds option for users to choose delay between individual feed synchronizations.
It was also mentioned in already closed #282. 
Why is there a delay in the first place anyway? I experimented with various values including 0 and saw no difference though I am aware that even 0 means non-zero due to setTimeout implementation. Git log just mentioned big update and correct scheduling.

tanriol commented:
These delays can probably be significantly reduced now. They used to be important when Brief used to run on the main thread of the only Firefox process. Now they still can slow down the extensions process, but that's much less noticeable (at least unless you have addons that use blocking webRequest or something like that). Still, reducing them to zero (especially the background one) is probably too much.

mmatous commented:
Updated to retain previous naming and changed all ^fetchDelay to ^defaultFetchDelay for consistency.
I don't think forgetting previous custom value was a problem though. If somebody messes with unexposed internals they can't really expect guarantees.
I left the minimum permissible delay at 0. It's been working well for me all this time so if anyone does encounter problems they can just not take the option.
I don't understand the concern about slow down. At least not in 2019. The requests are async, so are you worried about the parsing? I can't believe a few kB of well-formed XML is a problem either considering what the DOM parser has to suffer through on regular websites. Tested just on my ~30 subscribed feeds.
